homeassistant/components/config/config_entries.py

```diff
@@ -735,6 +736,49 @@ async def config_subentry_list(
     connection.send_result(msg["id"], result)
 
 
+@websocket_api.require_admin
+@websocket_api.websocket_command(
+    {
+        "type": "config_entries/subentries/update",
+        "entry_id": str,
+        "subentry_id": str,
+        vol.Optional("data"): Mapping,
+        vol.Optional("title"): str,
+        vol.Optional("unique_id"): str,
+    }
+)
+@websocket_api.async_response
+async def config_subentry_update(
+    hass: HomeAssistant,
+    connection: websocket_api.ActiveConnection,
+    msg: dict[str, Any],
+) -> None:
+    """Rename a subentry of a config entry."""
```

Comment:
The docstring does not match the name of the WS command or the name of the function. I think it's OK to name the function "update", although only the title can be changed at this point, so the docstring should be updated.